src/frontend/context/SecurityContext.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from "react";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+import { PASSWORD_KEY } from "../constants";
+
+type AppModeTypes = "normal" | "kill";
+type AuthStatusTypes = "pending" | "authenticated" | "notRequired";
```

Comment:
nit: but slightly redundant to suffix with `Types` here

Reply:
These are being used in a few places, so typing them just makes the coding a little easier

Reply:
think there's a misunderstanding here. was purely referring to the names of the type variables i.e. `AppModeTypes -> AppMode` and `AuthStatusTypes -> AuthStatus`